code/__DEFINES/dcs/signals/signals_fish.dm

```diff
@@ -22,6 +22,9 @@
 ///From /obj/item/fish/update_fish_force: (weight_rank, bonus_malus)
 #define COMSIG_FISH_FORCE_UPDATED "fish_force_updated"
 
+///From /obj/item/fish/interact_with_atom_secondary, sent to the target: (fish)
+#define TRAIT_FISH_RELEASED_INTO "fish_released_into"
```

Comment:
ghommie I love you but this is a signal not a trait. also what is fish if it has a typepath you should include it in the spec

Reply:
Totally didn't notice it, lol.